### ISSUE TITLE
When can we use ‘switch_to_gv’ method in the process of training？

### ISSUE THREAD

**Issue by JHW2000** [icandle/EFDN#4]
Thanks for your work! I've learned a lot and read your core code. Now I have some questions in training details.
In the paper, during the training phase we use EDBB with 7 branches. When training finished we reparameterize EDBB into vanilla convolution for inference and deployment. These can be implemented by 'switch_to_deploy' method in 'EFDN' class.
What confused me most is: 
1. what's the purpose of ‘switch_to_gv’ method in 'EFDN' class? As it discards conv1x1 and conv1x1_3x3 branches.
2. whether we need to use ‘switch_to_gv' during training phase?
Looking forward to your reply!

**Comment by icandle** [comment]
It was used to determined the \lambda_x, \lambda_y, \lambda_l of the EG loss. For simplicity, this part actually can be skipped by mannually set \lambda = 1e-3. That means it is not essential in the training process. But ‘switch_to_gv' might be work in the fine-tunning procedure (EDBB->simple EDBB->Conv), but we haven't try yet.
